packages/now-cli/src/util/once.ts

```diff
@@ -1,20 +0,0 @@
-import { EventEmitter } from 'events';
```

Comment:
What's the difference between `once.ts` and `@tootallnate/once`?

Reply:
Nothing, but since `@now/node` is now also using this module, I figured it would be better to keep the one copy in `node_modules` instead of >1 copy.

Reply:
Maybe it should be moved to `@now/build-utils` since thats where common code like this usually lives.

Reply:
I don't really think so. It's a generic function not related to Now specifically. Plus, I want to use it in `@zeit/fun` which we don't want to have `@now/build-utils` as a dependency.